src/renderer/components/pool/PoolDetails.tsx

```diff
@@ -61,13 +63,16 @@ export const PoolDetails: React.FC<Props> = ({
             isLoading={isLoading}
           />
         </A.Col>
+        <Styled.ColumnSpace />
```

Comment:
im not really sure why there is additional `div` element ? 🤔 
for small mobile views we can just do not drop bottom-margins for last 2 `Cards`. or do i miss anything ?
(screenshot was taken based on d36942f3cd82869cab320bb7502f1b2811c967a5 commit)
![image](https://user-images.githubusercontent.com/67793194/120230644-76bde880-c258-11eb-8064-ccc7f95c41ae.png)

Reply:
The extra `div` element is to create space between the cards and the chart on smaller views. But your suggested approach works as well and I like it. I've updated the code to go with your approach